libs/openFrameworks/types/ofParameter.h

```diff
@@ -3,6 +3,9 @@
 #include "ofEvents.h"
 #include "ofConstants.h"
 #include "ofPoint.h"
+#include "ofVec2f.h"
+#include "ofVec3f.h"
+#include "ofVec4f.h"
```

Comment:
Why do we need this? I'm trying to include as few headers as possible and don't remember this was needed here?

Reply:
there are some template specializations that need those, in the following lines.
https://github.com/openframeworks/openFrameworks/blob/master/libs/openFrameworks/types/ofParameter.h#L366-L393

I changed the includes in ofPoint.h which were indirectly including ofVec*f, so I thought that it was better to include this directly in this file rather than having it as a side effect of another include